index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Saul Greene Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="icon" type="image/x-icon"
+      href="assets/sauls-face.png"
+    >
+    <link rel="stylesheet" href="reset.css">
+  </head>
+  <header>
+    <h1>SAUL GREENE</h1>
+    <nav class="main-nav">
+      <span class="icon-menu"></span>
+      <ul>
+        <li><a href="#">ABOUT</a></li>
```

Comment:
Your nav items should be linking to Github and at least one social media page, i.e. LinkedIn, per instructions.